Source/santad/EventProviders/SNTEndpointSecurityFileAccessAuthorizer.mm

```diff
@@ -485,7 +485,7 @@ - (FileAccessPolicyDecision)handleMessage:(const Message &)msg
                               targetPathCopy, policyDecision);
                           }];
     }
-
+#if 0
```

Comment:
Is this intentional?

Reply:
Yes - since this is now configurable, i didn't want to actually pop open dialogs yet. So keeping it hardcoded as disabled.